### ISSUE TITLE
could not create environment using *.yml file

### ISSUE THREAD

**Issue by LianhaoYin** [conda/conda#4530]
after I use: conda env create -f environment.yml

following error happens, can someone give assistance, thank you very much !!

WARNING conda.lock:touch(53): Failed to create lock, do not run conda in parallel processes [errno 13]
An unexpected error has occurred.                  | ETA:  --:--:--   0.00  B/s
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Current conda install:

               platform : win-64
          conda version : 4.2.12
       conda is private : False
      conda-env version : 4.2.12
    conda-build version : not installed
         python version : 3.5.2.final.0
       requests version : 2.11.1
       root environment : C:\Program Files\Miniconda3  (read only)
    default environment : C:\Program Files\Miniconda3
       envs directories : C:\Users\Lianhao\.conda\envs
                          C:\Program Files\Miniconda3\envs
          package cache : C:\Users\Lianhao\.conda\envs\.pkgs
                          C:\Program Files\Miniconda3\pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : None
           offline mode : False

`$ C:\Program Files\Miniconda3\Scripts\conda-env-script.py create -f environment.yml`

    Traceback (most recent call last):
      File "C:\Program Files\Miniconda3\lib\site-packages\conda\fetch.py", line 429, in download
        with open(pp, 'wb') as fo:
    PermissionError: [Errno 13] Permission denied: 'C:\\Program Files\\Miniconda3\\pkgs\\wheel-0.29.0-py35_0.tar.bz2.part'

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "C:\Program Files\Miniconda3\lib\site-packages\conda_env\installers\conda.py", line 33, in install
        plan.execute_actions(actions, index, verbose=not args.quiet)
      File "C:\Program Files\Miniconda3\lib\site-packages\conda\plan.py", line 643, in execute_actions
        inst.execute_instructions(plan, index, verbose)
      File "C:\Program Files\Miniconda3\lib\site-packages\conda\instructions.py", line 134, in execute_instructions
        cmd(state, arg)
      File "C:\Program Files\Miniconda3\lib\site-packages\conda\instructions.py", line 47, in FETCH_CMD
        fetch_pkg(state['index'][arg + '.tar.bz2'])
      File "C:\Program Files\Miniconda3\lib\site-packages\conda\fetch.py", line 363, in fetch_pkg
        download(url, path, session=session, md5=info['md5'], urlstxt=True)
      File "C:\Program Files\Miniconda3\lib\site-packages\conda\fetch.py", line 450, in download
        raise CondaRuntimeError("Could not open %r for writing (%s)." % (pp, e))
    conda.exceptions.CondaRuntimeError: Runtime error: Could not open 'C:\\Program Files\\Miniconda3\\pkgs\\wheel-0.29.0-py35_0.tar.bz2.part' for writing ([Errno 13] Permission denied: 'C:\\Program Files\\Miniconda3\\pkgs\\wheel-0.29.0-py35_0.tar.bz2.part').

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "C:\Program Files\Miniconda3\lib\site-packages\conda\exceptions.py", line 479, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "C:\Program Files\Miniconda3\lib\site-packages\conda_env\cli\main_create.py", line 111, in execute
        installer.install(prefix, pkg_specs, args, env)
      File "C:\Program Files\Miniconda3\lib\site-packages\conda_env\installers\conda.py", line 40, in install
        raise CondaSystemExit('Exiting', e)
      File "C:\Program Files\Miniconda3\lib\contextlib.py", line 77, in __exit__
        self.gen.throw(type, value, traceback)
      File "C:\Program Files\Miniconda3\lib\site-packages\conda\cli\common.py", line 573, in json_progress_bars
        yield
      File "C:\Program Files\Miniconda3\lib\site-packages\conda_env\installers\conda.py", line 38, in install
        raise CondaRuntimeError('RuntimeError: %s' % e)
    conda.exceptions.CondaRuntimeError: Runtime error: RuntimeError: Runtime error: Could not open 'C:\\Program Files\\Miniconda3\\pkgs\\wheel-0.29.0-py35_0.tar.bz2.part' for writing ([Errno 13] Permission denied: 'C:\\Program Files\\Miniconda3\\pkgs\\wheel-0.29.0-py35_0.tar.bz2.part').

**Comment by kalefranz** [comment]
Please update to conda 4.3 using `conda update conda`, and file a new issue if you still encounter your problem.
